editor/src/components/inspector/common/css-utils.ts

```diff
@@ -762,7 +768,9 @@ export function printArrayCSSNumber(array: Array<GridCSSNumber>): string {
   return array
     .map((dimension) => {
       const printed = printCSSNumber(dimension, null)
-      return typeof printed === 'string' ? printed : `${printed}`
+      const areaName = dimension.areaName != null ? `[${dimension.areaName}] ` : ''
+      const value = typeof printed === 'string' ? printed : `${printed}`
```

Comment:
minor - why do we need to use the string template here? can we just pass `printed `to
`${areaName}${printed}` next line?

Reply:
sounds good https://github.com/concrete-utopia/utopia/pull/6080/commits/f5573860c5356073d009f153df9ea3a60b7b7b9a